instrumentation/camel-2.20/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/apachecamel/aws/S3CamelTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apachecamel.aws;
+
+import static io.opentelemetry.api.trace.SpanKind.CLIENT;
+import static io.opentelemetry.api.trace.SpanKind.CONSUMER;
+
+import com.amazonaws.services.sqs.model.PurgeQueueInProgressException;
+import com.google.common.collect.ImmutableMap;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Disabled("Does not work with localstack - X-Ray features needed")
+public class S3CamelTest {
+
+  @RegisterExtension
+  public static final InstrumentationExtension testing = AgentInstrumentationExtension.create();
+
+  private static final Logger logger = LoggerFactory.getLogger(S3CamelTest.class);
+
+  // To account for any delay interacting with real AWS environment
+  private static final int sqsDelay = 10000;
+
+  private static AwsConnector awsConnector;
+
+  @BeforeAll
+  protected static void setUp() {
+    awsConnector = AwsConnector.liveAws();
+  }
+
+  private static void waitAndClearSetupTraces(
```

Comment:
Nit: can you move it below the test method, so that you can read the class top-bottom?